screens/Register.js

```diff
@@ -1,33 +1,31 @@
 import React from 'react';
```

Comment:
Not sure why you have any changes in this file. Make sure that this file matches the `master` branch's version